.ebextensions/00_ami.config

```diff
@@ -0,0 +1,82 @@
+#
+# Nessie's AWS configs
+#
+packages:
+  yum:
+    gcc-c++: []
+    git: []
+    mod24_ssl: []
+    openssl-devel: []
+
+option_settings:
+  aws:autoscaling:launchconfiguration:
+    EC2KeyName: cloudlrs-aws-eb
+    InstanceType: t2.small
+    IamInstanceProfile: aws-elasticbeanstalk-ec2-role
```

Comment:
@johncrossman Nessi needs to have this security group config to SSH in custom VPC    
SSHSourceRestriction: tcp, 22, 22, sg-8ece7af1